packages/turf-nearest-point-to-line/index.d.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference types="geojson" />
+
+type Points = GeoJSON.FeatureCollection<GeoJSON.Point>;
+type LineString = GeoJSON.LineString;
```

Comment:
This would only allow Geometry LineString and not a `Feature<LineString>`:

**Correct definition for both Geometry & Feature support:**
```ts
type LineString = GeoJSON.Feature<GeoJSON.LineString> | GeoJSON.LineString;
```